plank.xcodeproj/project.pbxproj

```diff
@@ -172,7 +172,9 @@
 				OBJ_23 /* Utility */,
 				OBJ_24 /* Products */,
 			);
+			indentWidth = 4;
 			sourceTree = "<group>";
+			tabWidth = 4;
```

Comment:
Just a little ride along :)

Reply:
We may should have a different PR for it, but I would let @rahul-malik  make the call